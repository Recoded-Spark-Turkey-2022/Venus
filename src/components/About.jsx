import React from 'react'
// Import Swiper React components
import { Swiper, SwiperSlide } from "swiper/react";
import { Autoplay, Pagination } from "swiper";

import {StarIcon} from '@heroicons/react/24/outline'
import AboutImg from '../assets/about-image.svg'
import Button from './UI/Button'


import 'swiper/swiper-bundle.css';
import '../topics.css'





function About() {
  return (
    <div > 
        <div className='aboutfont flex flex-col'>
            <div className=' self-center'>
               <h2 className='text-center text-darkBlue font-bold text-[28px] mt-[43px]'>About</h2> 
               <p className=' text-center text-[14px] max-w-[637px] p-[7px] leading-5'>In quo quaerimus, non emolumento aliquo, sed ipsius honestatis decore laudandis, id est et quas molestias excepturi sint, obcaecati cupiditate non numquam eius modi tempora incidunt, ut earum rerum hic tenetur a natura ipsa natura incorrupte atque integre iudicante itaque aiunt hanc quasi involuta aperiri.</p>
            </div>
            <div className=' self-center'>
            <Button >Learn more</Button> 

            </div>
        </div>
        <div className='flex flex-col-reverse self-center lg:flex-row w-full '>
            <div className='w-1/2 '>
            
                    <Swiper
                      centeredSlides="true"
                      autoplay={{
                        delay: 2500,
                        disableOnInteraction: false,
                      }}
                     
                      pagination={{

                        clickable: true,
                        }}
                      
                      modules={[Autoplay, Pagination]}
                     className="mySwiper  lg:w-full"
                    >
                     <SwiperSlide><div>
                        <div className='flex'>
                            <StarIcon className='w-7 mr-4'/>
            
                            <h5 className='font-bold text-[16px] p-[13px]'>Reliability</h5>
                        </div>
                        <p className='text-left max-w-[224px]' >Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                    </div>
                    </SwiperSlide>
                     <SwiperSlide>
                        <div>
                          <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Motivation</h5>
                        </div>
                        <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                        </div>
                    </SwiperSlide>
                     <SwiperSlide>
                        <div>
                         <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Efficiency</h5>
                         </div> 
                         <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                    </div>
                    </SwiperSlide>
                    <SwiperSlide>
                        <div>
                        <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Creativity</h5>
                        </div>
                        <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                    </div> 
                    </SwiperSlide>
                    
                    </Swiper>
            </div>
            <div className='flex w-1/2 justify-center' >
                <img className=' ' src={AboutImg} alt='/' />
            </div>
        </div>
    </div>
  )
}

export default About