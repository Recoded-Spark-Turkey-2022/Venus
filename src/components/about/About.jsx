import React from 'react'

import { Swiper, SwiperSlide } from "swiper/react";
import { Autoplay, Pagination } from "swiper";
import { StarIcon } from '@heroicons/react/24/outline'
import AboutImg from '../../assets/about-image.svg'
import Button from '../UI/Button'

import 'swiper/swiper-bundle.css';
import "./about.css";





function About() {
  return (
    <div > 
        <div className='aboutfont flex flex-col'>
            <div className=' self-center'>
               <h2 className='text-center text-darkBlue font-bold text-[28px] mt-[43px]'>About</h2> 
               <p className=' text-center text-[14px] max-w-[637px] p-[7px] leading-5'>In quo quaerimus, non emolumento aliquo, sed ipsius honestatis decore laudandis, id est et quas molestias excepturi sint, obcaecati cupiditate non numquam eius modi tempora incidunt, ut earum rerum hic tenetur a natura ipsa natura incorrupte atque integre iudicante itaque aiunt hanc quasi involuta aperiri.</p>
            </div>
            <div className=' self-center'>
            <Button >Learn more</Button> 

            </div>
        </div>
        <div className='flex flex-col-reverse self-center lg:flex-row w-full flex justify-center'>
            <div className='lg:block self-center hidden '>
                <div className='flex '>
                        <div className='p-5'>
                          <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Motivation</h5>
                          </div>
                            <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                        </div>
                        <div className='p-5'>
                        <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Efficiency</h5>
                         </div> 
                         <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                        </div>
                </div>
                <div className='flex'>
                  <div className='p-5'>
                        <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Creativity</h5>
                        </div>
                        <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                  </div>
                  <div className='p-5'>
                        <div className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Reliability</h5>
                        </div>
                        <p className='text-left max-w-[224px]' >Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                  </div>
                </div>
            </div>

            <div className='about-swiper block lg:hidden  bg-inlightBlue w-full lg:w-1/2 '>
            
                    <Swiper
                      centeredSlides="true"
                      autoplay={{
                        delay: 2500,
                        disableOnInteraction: false,
                      }}
                     
                      pagination={{

                        clickable: true,

                        }}
                      
                      modules={[Autoplay, Pagination]}
                     className="about-swiper mySwiper lg:w-full"
                    >
                     <SwiperSlide>
                      <div>
                        <div className='flex'>
                            <StarIcon className='w-7 mr-4'/>
            
                            <h5 className='font-bold text-[16px] p-[13px]'>Reliability</h5>
                        </div>
                        <p className='text-left max-w-[224px]' >Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                    </div>
                    </SwiperSlide>
                     <SwiperSlide>
                        <div>
                          <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Motivation</h5>
                        </div>
                        <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                        </div>
                    </SwiperSlide>
                     <SwiperSlide>
                        <div>
                         <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Efficiency</h5>
                         </div> 
                         <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                    </div>
                    </SwiperSlide>
                    <SwiperSlide>
                        <div>
                        <div  className='flex'>
                            <StarIcon className='w-7 mr-4'/>
                            <h5 className='font-bold text-[16px] p-[13px]'>Creativity</h5>
                        </div>
                        <p className='text-left max-w-[224px]'>Certe, inquam, pertinax non recusandae itaque earum 	rerum facilis est consec. Laudem et impetus quo aut in gravissimo bello animadversionis.</p>
                    </div> 
                    </SwiperSlide>
                    
                    </Swiper>
            </div>
            <div className=' flex justify-center  w-full lg:w-1/2' >
                <img className=' w-[500px] lg:w-full ' src={AboutImg} alt='/' />
            </div>
        </div>
    </div>
  )
}

export default About