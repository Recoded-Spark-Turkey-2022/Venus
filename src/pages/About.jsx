import React from 'react';
import Cover from '../components/aboutPage/Cover';
import Share from '../components/aboutPage/Share';
import OurStory from '../components/aboutPage/OurStory';
import OurTeamSection from '../components/aboutPage/OurTeamSection';
import Container from '../components/UI/Container';
import ReferenceSection from '../components/UI/ReferenceSection';
import Partners from '../components/Partners/Partners';

const About = () => {
  return (
    <>
      <Cover />
      <Container>
        <OurStory />
      </Container>
      <OurTeamSection />
      <ReferenceSection/>
      <Container>
        <Partners/>
        <Share />
      </Container>
    </>
  );
};

export default About;
