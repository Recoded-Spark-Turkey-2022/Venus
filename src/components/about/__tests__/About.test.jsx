import React from 'react';
import renderer from "react-test-renderer";
import About from '../About';

jest.mock('swiper/react', () => ({
    Swiper: 'Swiper',
    SwiperSlide: 'SwiperSlide'
  }));
  
  jest.mock('swiper', () => ({
    Pagination: 'Pagination'
  }));

it(`renders correctly` , () => {
    const tree = renderer.create(<About />);
    expect(tree).toMatchSnapshot();
});