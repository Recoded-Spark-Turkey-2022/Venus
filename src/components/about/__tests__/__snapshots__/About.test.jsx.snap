// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="aboutfont flex flex-col"
  >
    <div
      className=" self-center"
    >
      <h2
        className="text-center text-darkBlue font-bold text-[28px] mt-[43px]"
      >
        Home.about
      </h2>
      <p
        className=" text-center text-[17px] max-w-[800px] p-[7px] leading-5"
      >
        Home.about-p
      </p>
    </div>
    <div
      className=" self-center"
    >
      <a
        href="/about"
        onClick={[Function]}
      >
        <button
          className="my-9 px-6 rounded-2xl  py-1"
          id="mediumBlue-button"
          type="button"
        >
           
          Button.learn
        </button>
         
      </a>
    </div>
  </div>
  <div
    className="flex flex-col-reverse self-center lg:flex-row w-full flex justify-center"
  >
    <div
      className="lg:block self-center hidden "
    >
      <div
        className="flex "
      >
        <div
          className="p-5"
        >
          <div
            className="flex"
          >
            <svg
              aria-hidden="true"
              className="w-7 mr-4"
              fill="none"
              stroke="currentColor"
              strokeWidth={1.5}
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
            <h5
              className="font-bold text-[16px] p-[13px]"
            >
              Home.p1
            </h5>
          </div>
          <p
            className="text-left max-w-[224px]"
          >
            Home.p1-p
          </p>
        </div>
        <div
          className="p-5"
        >
          <div
            className="flex"
          >
            <svg
              aria-hidden="true"
              className="w-7 mr-4"
              fill="none"
              stroke="currentColor"
              strokeWidth={1.5}
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
            <h5
              className="font-bold text-[16px] p-[13px]"
            >
              Home.p2
            </h5>
          </div>
          <p
            className="text-left max-w-[224px]"
          >
            Home.p2-p
          </p>
        </div>
      </div>
      <div
        className="flex"
      >
        <div
          className="p-5"
        >
          <div
            className="flex"
          >
            <svg
              aria-hidden="true"
              className="w-7 mr-4"
              fill="none"
              stroke="currentColor"
              strokeWidth={1.5}
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
            <h5
              className="font-bold text-[16px] p-[13px]"
            >
              Home.p3
            </h5>
          </div>
          <p
            className="text-left max-w-[224px]"
          >
            Home.p3-p
          </p>
        </div>
        <div
          className="p-5"
        >
          <div
            className="flex"
          >
            <svg
              aria-hidden="true"
              className="w-7 mr-4"
              fill="none"
              stroke="currentColor"
              strokeWidth={1.5}
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
            <h5
              className="font-bold text-[16px] p-[13px]"
            >
              Home.p4
            </h5>
          </div>
          <p
            className="text-left max-w-[224px]"
          >
            Home.p4-p
          </p>
        </div>
      </div>
    </div>
    <div
      className="about-swiper block lg:hidden  bg-inlightBlue w-full lg:w-1/2 "
    >
      <Swiper
        autoplay={
          Object {
            "delay": 2500,
            "disableOnInteraction": false,
          }
        }
        centeredSlides="true"
        className="about-swiper mySwiper lg:w-full"
        modules={
          Array [
            undefined,
            "Pagination",
          ]
        }
        pagination={
          Object {
            "clickable": true,
          }
        }
      >
        <SwiperSlide>
          <div>
            <div
              className="flex"
            >
              <svg
                aria-hidden="true"
                className="w-7 mr-4"
                fill="none"
                stroke="currentColor"
                strokeWidth={1.5}
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                />
              </svg>
              <h5
                className="font-bold text-[16px] p-[13px]"
              >
                Home.p1
              </h5>
            </div>
            <p
              className="text-left max-w-[224px]"
            >
              Home.p1-p
            </p>
          </div>
        </SwiperSlide>
        <SwiperSlide>
          <div>
            <div
              className="flex"
            >
              <svg
                aria-hidden="true"
                className="w-7 mr-4"
                fill="none"
                stroke="currentColor"
                strokeWidth={1.5}
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                />
              </svg>
              <h5
                className="font-bold text-[16px] p-[13px]"
              >
                Home.p2
              </h5>
            </div>
            <p
              className="text-left max-w-[224px]"
            >
              Home.p2-p
            </p>
          </div>
        </SwiperSlide>
        <SwiperSlide>
          <div>
            <div
              className="flex"
            >
              <svg
                aria-hidden="true"
                className="w-7 mr-4"
                fill="none"
                stroke="currentColor"
                strokeWidth={1.5}
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                />
              </svg>
              <h5
                className="font-bold text-[16px] p-[13px]"
              >
                Home.p3
              </h5>
            </div>
            <p
              className="text-left max-w-[224px]"
            >
              Home.p3-p
            </p>
          </div>
        </SwiperSlide>
        <SwiperSlide>
          <div>
            <div
              className="flex"
            >
              <svg
                aria-hidden="true"
                className="w-7 mr-4"
                fill="none"
                stroke="currentColor"
                strokeWidth={1.5}
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11.48 3.499a.562.562 0 011.04 0l2.125 5.111a.563.563 0 00.475.345l5.518.442c.499.04.701.663.321.988l-4.204 3.602a.563.563 0 00-.182.557l1.285 5.385a.562.562 0 01-.84.61l-4.725-2.885a.563.563 0 00-.586 0L6.982 20.54a.562.562 0 01-.84-.61l1.285-5.386a.562.562 0 00-.182-.557l-4.204-3.602a.563.563 0 01.321-.988l5.518-.442a.563.563 0 00.475-.345L11.48 3.5z"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                />
              </svg>
              <h5
                className="font-bold text-[16px] p-[13px]"
              >
                Home.p4
              </h5>
            </div>
            <p
              className="text-left max-w-[224px]"
            >
              Home.p4-p
            </p>
          </div>
        </SwiperSlide>
      </Swiper>
    </div>
    <div
      className=" flex justify-center  w-full lg:w-1/2"
    >
      <img
        alt="/"
        className=" w-[500px] lg:w-full "
        src="about-image.svg"
      />
    </div>
  </div>
</div>
`;
